src/utils/mtev_skiplist.c

```diff
@@ -227,6 +229,9 @@ void *mtev_skiplist_previous(mtev_skiplist *sl, mtev_skiplist_node **iter) {
   if(!(*iter)->prev) *iter = NULL;
   return (*iter)?((*iter)->data):NULL;
 }
+void *mtev_skiplist_data(mtev_skiplist_node *m) {
+  return m->data;
+}
```

Comment:
Does this accessor allow us to make the skiplist node structure opaque?  Just a question,  not a request to do that.